src/home/fetch-github/fetch-issues-full.ts

```diff
@@ -20,9 +20,13 @@ export async function postLoadUpdateIssues() {
     const fetchedIssues = await fetchIssues();
 
     if (issuesAreDifferent(cachedIssues, fetchedIssues)) {
-      await saveIssuesToCache(cachedIssues, fetchedIssues);
+      await saveIssuesToCache(cachedIssues, fetchedIssues); // this handles stale and new issues
       await taskManager.syncTasks();
-      void displayGitHubIssues();
+      if (cachedIssues) {
+        void displayGitHubIssues(undefined, undefined, true); // if there were cached issues skip animation
```

Comment:
I think it makes sense to change the function signature so that you don't have to pass in two undefined? 

Alternatively, if it makes sense, pass in destructured parameters (an object)